de.danielnoethen.butt.json

```diff
@@ -0,0 +1,108 @@
+{
+    "app-id": "de.danielnoethen.butt",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "butt",
+    "rename-icon": "butt",
+    "rename-desktop-file": "butt.desktop",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-run/pipewire-0",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.mpris.MediaPlayer2.*",
+        "--talk-name=org.gnome.Rhythmbox3"
```

Comment:
what is `org.gnome.Rhythmbox3` for?

Reply:
Used to communicate to gnomes rhythm box over dbus. 
It's used by the application to get now playing information.

Reply:
> Used to communicate to gnomes rhythm box over dbus.

Why rhythmbox only? Seems a bit weird. Does it not work with other apps?

Reply:
Yeah, it works with other applications via the other dbus talk name.